trunk/NDHMS/Routing/Noah_distr_routing_subsurface.F

```diff
@@ -204,7 +208,6 @@ SUBROUTINE SUBSFC_RTNG(subrt_data, subrt_static, subrt_input, subrt_output, CWAT
 
     !DJG Set up mass balance checks...
     !         CWATAVAIL = 0.            !-- initialize subsurface watavail
-    SUBDT = subrt_static%dt                !-- initialize the routing timestep to DT
```

Comment:
@zhengtaocui, can you give a brief reason as to why you commented out this line?

Reply:
@rcabell This line was moved up to prevent crashes when HYDRO_D is enabled. There is a comment in the file where the line is moved to.
    !Z. Cui -----------------------------------------------------------------
    !Move SUBDT initialization up to avoid crash when HYDRO_D is defined
    SUBDT = subrt_static%dt                !-- initialize the routing timestep to DT
    !Z. Cui -----------------------------------------------------------------

Reply:
Great, I see now. Thanks.